source/configuration_manual/mail_location/index.rst

```diff
@@ -1,299 +1,367 @@
+``VOLATILEDIR``    Specifies the location of volatile files. This includes
+                   lock files and potentially other files that don't need to
+                   exist permanently. This is especially useful to avoid
+                   creating lock files to NFS or other remote filesystems.
+                   (v2.2.32+)
 
-UTF-8               Store mailbox names on disk using UTF-8 instead of modified UTF-7.
+``SUBSCRIPTIONS``  Specifies the file used for storing subscriptions. The
+                   default is ``subscriptions``. If you're trying to avoid
+                   name collisions with a mailbox named ``subscriptions``,
+                   then also consider setting ``MAILBOXDIR``.
 
-BROKENCHAR          Specifies an escape character that is used for broken or otherwise inaccessible mailbox names. If mailbox name can't be changed reversibly to UTF-8 and back, encode the problematic parts using ``<broken_char><hex>`` in the user-visible UTF-8 name. The broken_char itself also has to be encoded the same way. This can be useful with :ref:`imapc_mbox_format` to access mailbox names that aren't valid mUTF-7 charset from remote servers, or if the remote server uses a different hierarchy separator and has folder names containing the local separator. (v2.2.32+)
+``MAILBOXDIR``     Specifies directory name under which all mailbox
+                   directories are stored. The default is empty unless
+                   otherwise described in the mailbox format pages.
 
-		    .. versionchanged:: v2.3.14 Conflicting separators are also escaped.
+``FULLDIRNAME``    Specifies the directory name used for mailbox, index, and
+                   control directory paths. See the individual mailbox format
+                   pages for further information. This key replaced the
+                   deprecated ``DIRNAME`` key.  (v2.2.8+)
 
-CONTROL             Specifies the location of control files under the :ref:`mbox <mbox_settings_control_files>` or `Maildir <maildir_settings_control_files>` formats.
+``ALT``            Specifies the
+                   :ref:`alternate storage <dbox_settings_alt_storage>` path.
+================== =============================================================
+
+Deprecated Keys
+---------------
+
+============ ================ ==============================================
+Key          Replaced By      Value Description
+============ ================ ==============================================
+``DIRNAME``  ``FULLDIRNAME``  Specifies the directory name used for mailbox
+                              directories, or in the case of mbox specifies
+                              the mailbox message file name. Deprecated in
+                              v2.2.8.
```

Comment:
"Note that DIRNAME is not used for index or control directories, while FULLDIRNAME is."